src/components/DonutChart.js

```diff
@@ -80,19 +81,23 @@ const DonutChart = React.createClass({
   },
 
   _handleMouseEnter (index) {
-    this.setState({
-      activeIndex: index
-    });
+    if (this.props.animateOnHover) {
+      this.setState({
+        activeIndex: index
+      });
 
-    this.props.onMouseEnter(index);
+      this.props.onMouseEnter(index);
```

Comment:
Wouldn't we at least want to call this prop if animation is off? What if a user wants to get the value of the hovered slice and do something with it?